apps/frontend-pwa/src/pages/course/[courseId]/element/[id].tsx

```diff
@@ -323,7 +371,86 @@ function LearningElement({ courseId, id }: Props) {
                           <div className="font-bold">Multiplikator</div>
                           <div>{currentInstance.pointsMultiplier}x</div>
                         </div>
-                        <FontAwesomeIcon icon={faFlag}></FontAwesomeIcon>
+                        <Modal
+                          className={{ content: 'm-10 md:m-20' }}
```

Comment:
We should style the modal a bit more to only take the space it needs and still work nicely on mobile devices. Probably this will also need testing in production later on.

![Screenshot 2023-02-17 at 16 44 12](https://user-images.githubusercontent.com/80708107/219699788-e06c2ca4-480b-48b4-90ef-14f691d6c98f.png)

Reply:
resolved